plugins/generator-1.19.4/forge-1.19.4/livingentity.definition.yaml

```diff
@@ -24,4 +24,7 @@ localizationkeys:
     suffix: " Spawn Egg"
     condition: hasSpawnEgg
   - key: entity.@modid.@registryname
-    mapto: mobName
\ No newline at end of file
+    mapto: mobName
+  - key: entity.@modid.@registryname.none

```

Comment:
Where is this key used?

Reply:
I see, but then the condition is not complete

Reply:
Oh no, nvm, I was mistaken. This is used in the entity's trading GUI to show the entity's name. It is both for Villager and Wandering Trader trading types. It has `none` at the end because even as Villager trading type, there's no profession block, so no "profession", only trades and levels.